tools/oio-rebalance-rawx.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2015-2018 OpenIO SAS, as part of OpenIO SDS
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import shlex
+import subprocess
+from oio.common.json import json as jsonlib
+
+def execute(cmd, stdin=None):
```

Comment:
This function already exists in [tests/functional/cli/__init__.py](tests/functional/cli/__init__.py). Maybe we can move it to [oio/cli/__init__.py](oio/cli/__init__.py) to avoid duplicating it.